docs/components/lists.md

```diff
@@ -0,0 +1,452 @@
+<!--docs:
+title: "Lists"
+layout: detail
+section: components
+excerpt: "Lists are continuous, vertical indexes of text or images."
+iconId:
+path: /catalog/lists/
+-->
+
+# Lists
+
+[Lists](https://material.io/components/lists/) are continuous, vertical indexes of text or images.
+
+There are three list types:
+1. [Single-line list](#single-line-list)
+1. [Two-line list](#two-line-list)
+1. [Three-line list](#three-line-list)
+
+![Composite image of the three list types](assets/lists/lists-types.png)
+
+## Using lists
+
+### Making lists accessible
+
+Flutter's card component APIs support labeling for accessibility. For more guidance on writing labels, go to [our page on how to write a good accessibility label](https://material.io/design/usability/accessibility.html#writing).
+
+
+## List anatomy
+
+The following anatomy diagram applies to all three list types:
+
+![List anatomy diagram showing list, row, and list item content](assets/lists/list-content-anatomy.png)
+
+The following anatomy diagrams apply  to the list item content:
+
+![List item content showing supporting visuals, primary text, metadata, and controls in two list components](assets/lists/combined-list-content.png)
+
+
+A list consists of the following attributes:
+1. List
+2. Row
+3. List item content
+    1. Supporting visuals
+    2. Primary text
+    3. Metadata
+    4. Controls
+
+
+
+## Single-line list
+
+Single-line list items contain a maximum of one line of text.
+
+
+
+### Single-line list example
+
+Source code API:
+`ListTile`
+* [Class definition](https://api.flutter.dev/flutter/material/ListTile-class.html)
+* [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/list_tile.dart)
+* [Dartpad Demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/lists/single)
+
+![Single-line list example](assets/lists/single-line-list-example.png)
+
+```dart
+ListView(
+  children: [
+    ListTile(
+      title: Text('Primary text'),
+      leading: Icon(Icons.label),
+      trailing: Text('Metadata'),
+    ),
+  ],
+)
+```
+
+### Single-line list key properties
+
+
+#### List properties
+
+| &nbsp; | Property |
+|---|---|
+| Padding between rows | `padding` on `ListView` |
```

Comment:
padding on ListView doesn't actually adjust the padding between rows. It just adds the padding around the whole list

This would have to be done by either:

- the `contentPadding` property on `ListTile`
- inserting children to act as padding in between `ListTile`s

Reply:
Good call. I'll update the docs.